databuilder/publisher/elasticsearch_constants.py

```diff
@@ -40,11 +42,13 @@
             },
             "description": {
               "type": "text",
-              "analyzer": "simple"
+              "analyzer": "simple",
+              "search_analyzer": "whitespace",
             },
             "column_names": {
               "type":"text",
               "analyzer": "simple",
+              "search_analyzer": "whitespace",
```

Comment:
I found that using this combination of simple for the analyzer and whitespace for the search analyzer meant that we never got matches for column names with underscores in them, so ended up just switching this to 
```
"column_names": {
              "type":"text",
              "analyzer": "whitespace",
              "fields": ...
```

The same would probably be true for database below, too.

Reply:
thanks @jdavidheiser , pr updated